package/Dockerfile

```diff
@@ -46,7 +46,7 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=100.0.2+up0.3.8
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=1.0.2+up0.2.2
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=1.0.3+up0.2.4-rc2
```

Comment:
Doesn't prerelease go before metadata usually? 
https://regex101.com/r/1vI80U/1 - using regex from https://semver.org/

Reply:
This is the way rancher-charts works.

Reply:
We are releasing a new version of the chart (1.0.3) and the image contained in this version is 0.2.4-rc2. Once the chart is released as part of rancher/charts release, then the version of the image will be 0.2.4